spec/factories.rb

```diff
@@ -19,3 +19,12 @@
   c.redirect_uri  'https://client.example.com/cb'
   c.owner
 end
+
+Factory.define :native_client, :class => OAuth2::Model::Client do |c|
+  c.client_id     { OAuth2::Lib::SecureCodeScheme.random_string }
+  c.client_secret { OAuth2::Lib::SecureCodeScheme.random_string }
+  c.name          { Factory.next :client_name }
+  c.redirect_uri  'flickapp://client.example.com/cb'
```

Comment:
Don't use flick as example in specs, these changes may be PR'd to the original repo